CONVERT_SERVICE/convert_service/convert_app/views.py

```diff
@@ -0,0 +1,48 @@
+#
+# @views.py Copyright (c) 2021 Jalasoft.
+# 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+# All rights reserved.
+#
+# This software is the confidential and proprietary information of
+# Jalasoft, ("Confidential Information").  You shall not
+# disclose such Confidential Information and shall use it only in
+# accordance with the terms of the license agreement you entered into
+# with Jalasoft.
+#
+
+from django.core.files.storage import FileSystemStorage
+from pathlib import Path
+from django.views import View
+from django.http import HttpResponse
+from .model.ffmpeg.ffmpeg_execute import ffmpegexecute
+from .model.ffmpeg.img_compresor import zip_dir
+import json
+
+
+# Create your views here.
+
+
```

Comment:
Only two blank lines between the import statements and the code. Check this link:
https://www.python.org/dev/peps/pep-0008/#blank-lines